google-cloud-pubsub/lib/google/cloud/pubsub/project.rb

```diff
@@ -144,10 +144,10 @@ def topic topic_name, project: nil, skip_lookup: nil
         #   pubsub = Google::Cloud::Pubsub.new
         #   topic = pubsub.create_topic "my-topic"
         #
-        def create_topic topic_name
+        def create_topic topic_name, async: nil
```

Comment:
Project#topic doesn't accept async options, only create_topic does. Which means I can't do the following:

```ruby
topic = pubsub.topic "my-topic", max_messages: 50
```

😢